graftm/clusterer.py

```diff
@@ -7,11 +7,11 @@
 
 class Clusterer:
 
-    def __init__(self, slash_endings):
+    def __init__(self, has_slash_endings):
         '''
         Parameters
         ----------
-        slash_endings : bool
+        has_slash_endings : bool
```

Comment:
actually I'm confused. By making has_slash_endings an instance variable of Clusterer, are you assuming that all read sets either have or all read sets do not have, the slash endings? Might be good to test for this, too

Shouldn't has_slash_endings be derived from the Unpack, not the Clusterer?